src/languages/es.ts

```diff
@@ -433,6 +433,7 @@ const translations = {
         drafts: 'Borradores',
         finished: 'Finalizados',
         upgrade: 'Mejora',
+        downgrade: 'Degradar',
```

Comment:
I used these copies from Google Translate for this PR. Later, we’ll need finalized versions of the copies here

Reply:
@jayeshmangwani Please help to bring it to open-source channel

Reply:
Sure thing! I’ve asked for help here on [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q/p1730105986836259).